lua/copilot/client.lua

```diff
@@ -49,7 +49,7 @@ function M.get_node_version()
   if not M.node_version then
     local node = config.get("copilot_node_command")
 
-    local cmd = node .. " --version"
+    local cmd = { "node", "--version" }
```

Comment:
It should be `node` (the variable), not the string `"node"`:

```suggestion
    local cmd = { node, "--version" }
```